code/game/objects/items/reagent_containers/glass/bottle.dm

```diff
@@ -147,6 +147,12 @@
 	desc = "A small bottle. Contains hot sauce."
 	icon_state = "bottle3"
 	list_reagents = list(/datum/reagent/consumable/capsaicin = 60)
+	
+/obj/item/reagent_container/glass/bottle/dermaline
+	name = "\improper Dermaline bottle"
+	desc = "A small bottle. Contains dermaline - used as a potent treatment against burns."
+	icon_state = "bottle15"
+	list_reagents = list(/datum/reagent/medicine/dermaline = 60)	list_reagents = list(/datum/reagent/consumable/capsaicin = 60)
```

Comment:
```suggestion
	list_reagents = list(/datum/reagent/medicine/dermaline = 60, /datum/reagent/consumable/capsaicin = 60)
```

Reply:
spicy burn juice

Reply:
sonova. I knew I misclicked earlier, THAT's what it did.